sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -766,6 +766,26 @@ public async Task RecognizeLinkedEntitiesBatchWithErrorTest()
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
 
+        [Test]
+        public void RecognizeLinkedEntitiesBatchWithInvalidDocumentBatch()
```

Comment:
You can also add a test to ReconizeEntities right now (these two are the only endpoints with this deployed(

Reply:
great! added